shared/constants/metametrics.ts

```diff
@@ -586,7 +586,17 @@ export enum MetaMetricsEventName {
   UserClickedDeepLink = 'User Clicked Deeplink',
   UserClickedConnectCustodialAccount = 'Clicked Connect Custodial Account',
   UserClickedPortfolioButton = 'Clicked Portfolio Button',
+  UserClickedStakeButton = 'Clicked Stake Button',
   UserClickedCompliance = 'Clicked Compliance',
+  UserClickedRefreshTokenLink = 'User clicked refresh token link',
+  ShowDeeplinkForSignature = 'Show deeplink for signature',
+  CustodianOnboarding = 'Custodian onboarding',
+  InstitutionalFeatureConnection = 'Institutional feature connection',
+  CustodianSelected = 'Custodian selected',
+  ConnectCustodian = 'Connect to custodian',
+  ConnectCustodianCancel = 'Connect to custodian cancel',
+  ConnectCustodianError = 'Connect to custodian error',
+  CustodialAccountsConnected = 'Custodial accounts connected',
```

Comment:
Event names should all have title casing, and some of these don't make sense as actions. What does 'Custodian Onboarding' mean?. Has @worldlyjohn been asked to validate this schema?

Reply:
thanks @brad-decker .  i have not, but here are the relevant links.

- these events must be defined in our MM-wide data schema.  it's a new process that we just started implemented. [MetaMask: Working with Analytics/Events](https://docs.google.com/presentation/d/1HuMYZ8nVm-oLiyx8ds-JdcSFQ-_6Jcf2uWcLi43CtZ8/edit#slide=id.g22dbc521061_0_32) (Slide Deck)
- [Event Naming Conventions](https://www.notion.so/Event-Naming-Conventions-95bf27517033451888053f7ab654d327) (Consensys Notion)

Reply:
Hey folks, thanks for the feedback, I'll update the event names to be Title Casing, they are the ones we have in use already. My plan was to add/update them here in the extension and then add them in the [Segment Schema](https://github.com/ConsenSys/segment-schema) repo.
Also I don't have access to the [Event Naming Conventions](https://www.notion.so/Event-Naming-Conventions-95bf27517033451888053f7ab654d327) (Consensys Notion) can you provide it to me?